rewrite-gradle/src/main/java/org/openrewrite/gradle/UpdateDependencyLock.java

```diff
@@ -62,26 +65,32 @@ public boolean isAcceptable(SourceFile sourceFile, ExecutionContext executionCon
             return plainText;
         }
 
+        Arrays.stream(text.split("\n")).forEach(lock -> {
+            if (isComment(lock)) {
+                comments.add(lock);
+            } else {
+                parseLockedConfigurations(lock, lockedArtifacts, lockedConfigurations);
+            }
+        });
+
         GradleProject project = gradleProject.get();
         project.getConfigurations().stream()
                 .filter(GradleDependencyConfiguration::isCanBeResolved)
+                .filter(configuration -> lockedConfigurations.contains(configuration.getName()))
                 .forEach(conf -> {
                     List<ResolvedDependency> transitivelyResolvedDependencies = conf.getResolved();
+                    String configuration = conf.getName();
```

Comment:
It looks like we only ever use the configuration name, but in one lambda used `GradleDependencyConfiguration configuration`, while in the next assigned `String configuration`. Figured make that consistent through these changes:
```suggestion
                .map(GradleDependencyConfiguration::getName)
                .filter(lockedConfigurations::contains)
                .forEach(configuration -> {
                    List<ResolvedDependency> transitivelyResolvedDependencies = conf.getResolved();
```

Reply:
conf.getResolved() would not exist in this suggestion, would it?

Reply:
We need the dependencies in the configuration and its name.

Reply:
Ah I missed that one line, thanks!